server/routes/auth.js

```diff
@@ -16,5 +19,7 @@ auth.post(`${BASE_URL}/signup`, signUpValidation(), createUser);
 auth.post(`${BASE_URL}/login`, loginValidation(), login);
 // Verify User route
 auth.get(`${BASE_URL}/verify`, verifyToken, verifyUser);
+// pagination
+auth.get(`${BASE_URL}`, pagination);
```

Comment:
This is in the wrong file. This has nothing to do with auth.js

Reply:
ok